examples/nirfmxgsm/orfs.py

```diff
@@ -0,0 +1,315 @@
+2. Configure Frequency Reference.
+3. Configure External Attenuation.
+4. Configure Center Frequency.
+5. Configure Link Direction.
+6. Configure Trigger Parameters for IQ Power Edge Trigger.
+7. Configure Number of Timeslots.
+8. Configure Signal Type.
+9. Configure Auto Level.
+10. Configure Auto tsc Detection Enabled.
+11. Configure tsc.
+12. Select  ORFS  measurement and enable Traces.
+13. Configure Noise Compensation Enabled.
+14. Configure Measurement Type.
+15. Configure Offset Frequency Mode.
+16. Configure Evaluation Symbols.
+17. Configure Averaging Parameters.
+18. Initiate the Measurement.
+19. Fetch ORFS Measurements and Traces.
+20. Close RFmx Session.
+
+The gRPC API is built from the C API. RFmx GSM documentation is installed with the driver at:
+C:\Program Files (x86)\National Instruments\RFmx\GSM\Documentation\rfmxgsmcvi.chm
+
+Getting Started:
+
+To run this example, install "RFmx GSM" on the server machine.
+Link: https://www.ni.com/en-us/support/downloads/software-products/download.rfmx-gsm-edge-.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page.
+Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI-RFmx GSM gRPC Wiki for the latest C Function Reference:
+Link: https://github.com/ni/grpc-device/wiki/NI-RFmx-GSM-C-Function-Reference
+
+Running from command line:
+
+Server machine's IP address, port number, and physical channel name can be passed as separate
+command line arguments.
+  > python orfs.py <server_address> <port_number> <physical_channel_name>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763", with "SimulatedDevice" as the resource name
+"""
+
+import sys
+
+import grpc
+import nirfmxgsm_pb2 as nirfmxgsm_types
+import nirfmxgsm_pb2_grpc as grpc_nirfmxgsm
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+SESSION_NAME = "RFmxGSMSession"
+
+# Resource name and options for a simulated 5663 client.
+RESOURCE = "SimulatedDevice"
+OPTIONS = "Simulate=1,DriverSetup=Model:5663"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+if len(sys.argv) >= 4:
+    RESOURCE = sys.argv[3]
+    OPTIONS = ""
+
+# Create a gRPC channel + client.
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}")
+client = grpc_nirfmxgsm.NiRFmxGSMStub(channel)
+instr = None
+
+
+def check_for_warning(response, instrument):
+    """Print to console if the status indicates a warning."""
+    if response.status > 0:
+        warning_message = client.GetErrorString(
+            nirfmxgsm_types.GetErrorStringRequest(
+                instrument=instrument,
+                error_code=response.status,
+            )
+        )
+        sys.stderr.write(
+            f"{warning_message.error_description}\nWarning status: {response.status}\n"
+        )
+
+
+try:
```

Comment:
Same here, can you add the `except` for this `try` to catch API errors like what was done for WCDMA: https://github.com/ni/grpc-device/blob/ab6c951b449ab92e446a251cf52b06eb7f2dccef/examples/nirfmxwcdma/acp-single-carrier.py#L284